addons/account/tests/test_account_move_duplicate.py

```diff
@@ -87,8 +87,9 @@ def test_in_invoice_multiple_duplicate_reference_constrains(self):
         invoices.ref = invoice_1.ref
 
         # test constrains: batch without any previous posted invoice
-        with self.assertRaises(RedirectWarning):
+        with self.assertRaises(RedirectWarning) as cm:
```

Comment:
why "cm" ? :D

Reply:
It's from the documentation (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) 😅 Supposed to indicate "context manager".

Reply:
oh right, ok then